dts/bindings/interrupt-controller/ite,it8xxx2-wuc.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE Wake-Up Controller (WUC)
+compatible: "ite,it8xxx2-wuc"
+
+include: [base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  label:
+    required: true
+
+  "wakeup-controller":
+    type: boolean
+    required: true
+    description: convey's this node is an wakeup controller
+
+  "#wuc-cells":
+    type: int
+    required: true
+    description: number of items to expect in an wakeup controller specifier
+
+wuc-cells:
+  - group
```

Comment:
See above - the group # is redundant with the wuc phandle.

Reply:
Done